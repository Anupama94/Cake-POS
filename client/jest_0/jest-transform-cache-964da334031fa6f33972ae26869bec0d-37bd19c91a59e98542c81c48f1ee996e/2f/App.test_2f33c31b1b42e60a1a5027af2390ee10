37e896b19d3c8d0b145baf376dc92fbb
"use strict";

var _interopRequireDefault = require("/home/anupamaa/Desktop/Training/pos/server/client/node_modules/@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _App = _interopRequireDefault(require("./App"));

var _jsxFileName = "/home/anupamaa/Desktop/Training/pos/server/client/src/App.test.js";
describe('App', () => {
  it('should render correctly in "debug" mode', () => {
    const component = (0, _enzyme.shallow)(_react.default.createElement(_App.default, {
      debug: true,
      __source: {
        fileName: _jsxFileName,
        lineNumber: 22
      },
      __self: void 0
    })); // expect(component).toMatchSnapshot();

    expect(component).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbImRlc2NyaWJlIiwiaXQiLCJjb21wb25lbnQiLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7Ozs7QUFlQTs7QUFDQTs7QUFDQTs7O0FBRUFBLFFBQVEsQ0FBQyxLQUFELEVBQVEsTUFBTTtBQUNsQkMsRUFBQUEsRUFBRSxDQUFDLHlDQUFELEVBQTRDLE1BQU07QUFDaEQsVUFBTUMsU0FBUyxHQUFHLHFCQUFRLDZCQUFDLFlBQUQ7QUFBSyxNQUFBLEtBQUssTUFBVjtBQUFBO0FBQUE7QUFBQTtBQUFBO0FBQUE7QUFBQSxNQUFSLENBQWxCLENBRGdELENBR2pEOztBQUNDQyxJQUFBQSxNQUFNLENBQUNELFNBQUQsQ0FBTixDQUFrQkUsZUFBbEI7QUFDSCxHQUxDLENBQUY7QUFNSCxDQVBPLENBQVIiLCJzb3VyY2VzQ29udGVudCI6WyIvLyBpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuLy8gaW1wb3J0IFJlYWN0RE9NIGZyb20gJ3JlYWN0LWRvbSc7XG4vLyBpbXBvcnQgQXBwIGZyb20gJy4vQXBwJztcbi8vXG4vLyBpdCgncmVuZGVycyB3aXRob3V0IGNyYXNoaW5nJywgKCkgPT4ge1xuLy8gICBjb25zdCBkaXYgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdkaXYnKTtcbi8vICAgUmVhY3RET00ucmVuZGVyKDxBcHAgLz4sIGRpdik7XG4vLyAgIFJlYWN0RE9NLnVubW91bnRDb21wb25lbnRBdE5vZGUoZGl2KTtcbi8vIH0pO1xuXG5cbi8vIHRlc3QoJ0Zha2UgVGVzdCcsICgpID0+IHtcbi8vICAgICBleHBlY3QodHJ1ZSkudG9CZVRydXRoeSgpO1xuLy8gfSlcblxuaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCB7IHNoYWxsb3cgfSBmcm9tICdlbnp5bWUnO1xuaW1wb3J0IEFwcCBmcm9tICcuL0FwcCc7XG5cbmRlc2NyaWJlKCdBcHAnLCAoKSA9PiB7XG4gICAgaXQoJ3Nob3VsZCByZW5kZXIgY29ycmVjdGx5IGluIFwiZGVidWdcIiBtb2RlJywgKCkgPT4ge1xuICAgICAgICBjb25zdCBjb21wb25lbnQgPSBzaGFsbG93KDxBcHAgZGVidWcgLz4pO1xuXG4gICAgICAgLy8gZXhwZWN0KGNvbXBvbmVudCkudG9NYXRjaFNuYXBzaG90KCk7XG4gICAgICAgIGV4cGVjdChjb21wb25lbnQpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICAgIH0pO1xufSk7Il19