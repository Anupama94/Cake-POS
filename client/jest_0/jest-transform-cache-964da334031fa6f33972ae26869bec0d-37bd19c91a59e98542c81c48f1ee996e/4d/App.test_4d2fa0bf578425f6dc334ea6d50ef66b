f3435195532fa47f44a7620bb4f6b18b
"use strict";

var _interopRequireDefault = require("/home/anupamaa/Desktop/Training/pos/server/client/node_modules/@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _App = _interopRequireDefault(require("./App"));

var _jsxFileName = "/home/anupamaa/Desktop/Training/pos/server/client/src/App.test.js";
describe('App', () => {
  it('should render correctly in "debug" mode', () => {
    const component = (0, _enzyme.shallow)(_react.default.createElement(_App.default, {
      debug: true,
      __source: {
        fileName: _jsxFileName,
        lineNumber: 22
      },
      __self: void 0
    })); // expect(component).toMatchSnapshot();

    expect(toJson(component)).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbImRlc2NyaWJlIiwiaXQiLCJjb21wb25lbnQiLCJleHBlY3QiLCJ0b0pzb24iLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7Ozs7QUFlQTs7QUFDQTs7QUFDQTs7O0FBRUFBLFFBQVEsQ0FBQyxLQUFELEVBQVEsTUFBTTtBQUNsQkMsRUFBQUEsRUFBRSxDQUFDLHlDQUFELEVBQTRDLE1BQU07QUFDaEQsVUFBTUMsU0FBUyxHQUFHLHFCQUFRLDZCQUFDLFlBQUQ7QUFBSyxNQUFBLEtBQUssTUFBVjtBQUFBO0FBQUE7QUFBQTtBQUFBO0FBQUE7QUFBQSxNQUFSLENBQWxCLENBRGdELENBR2pEOztBQUNDQyxJQUFBQSxNQUFNLENBQUNDLE1BQU0sQ0FBQ0YsU0FBRCxDQUFQLENBQU4sQ0FBMEJHLGVBQTFCO0FBQ0gsR0FMQyxDQUFGO0FBTUgsQ0FQTyxDQUFSIiwic291cmNlc0NvbnRlbnQiOlsiLy8gaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0Jztcbi8vIGltcG9ydCBSZWFjdERPTSBmcm9tICdyZWFjdC1kb20nO1xuLy8gaW1wb3J0IEFwcCBmcm9tICcuL0FwcCc7XG4vL1xuLy8gaXQoJ3JlbmRlcnMgd2l0aG91dCBjcmFzaGluZycsICgpID0+IHtcbi8vICAgY29uc3QgZGl2ID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnZGl2Jyk7XG4vLyAgIFJlYWN0RE9NLnJlbmRlcig8QXBwIC8+LCBkaXYpO1xuLy8gICBSZWFjdERPTS51bm1vdW50Q29tcG9uZW50QXROb2RlKGRpdik7XG4vLyB9KTtcblxuXG4vLyB0ZXN0KCdGYWtlIFRlc3QnLCAoKSA9PiB7XG4vLyAgICAgZXhwZWN0KHRydWUpLnRvQmVUcnV0aHkoKTtcbi8vIH0pXG5cbmltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG5pbXBvcnQgeyBzaGFsbG93IH0gZnJvbSAnZW56eW1lJztcbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAnO1xuXG5kZXNjcmliZSgnQXBwJywgKCkgPT4ge1xuICAgIGl0KCdzaG91bGQgcmVuZGVyIGNvcnJlY3RseSBpbiBcImRlYnVnXCIgbW9kZScsICgpID0+IHtcbiAgICAgICAgY29uc3QgY29tcG9uZW50ID0gc2hhbGxvdyg8QXBwIGRlYnVnIC8+KTtcblxuICAgICAgIC8vIGV4cGVjdChjb21wb25lbnQpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICAgICAgICBleHBlY3QodG9Kc29uKGNvbXBvbmVudCkpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICAgIH0pO1xufSk7Il19