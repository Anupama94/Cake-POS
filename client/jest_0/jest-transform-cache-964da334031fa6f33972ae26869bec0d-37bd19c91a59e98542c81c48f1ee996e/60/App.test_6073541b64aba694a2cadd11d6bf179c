115d6de28ed17f8e40f54dcc48c71c26
"use strict";

// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
//
// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
test('Fake Test', () => {});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbInRlc3QiXSwibWFwcGluZ3MiOiI7O0FBQUE7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBR0FBLElBQUksQ0FBQyxXQUFELEVBQWMsTUFBTSxDQUV2QixDQUZHLENBQUoiLCJzb3VyY2VzQ29udGVudCI6WyIvLyBpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuLy8gaW1wb3J0IFJlYWN0RE9NIGZyb20gJ3JlYWN0LWRvbSc7XG4vLyBpbXBvcnQgQXBwIGZyb20gJy4vQXBwJztcbi8vXG4vLyBpdCgncmVuZGVycyB3aXRob3V0IGNyYXNoaW5nJywgKCkgPT4ge1xuLy8gICBjb25zdCBkaXYgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdkaXYnKTtcbi8vICAgUmVhY3RET00ucmVuZGVyKDxBcHAgLz4sIGRpdik7XG4vLyAgIFJlYWN0RE9NLnVubW91bnRDb21wb25lbnRBdE5vZGUoZGl2KTtcbi8vIH0pO1xuXG5cbnRlc3QoJ0Zha2UgVGVzdCcsICgpID0+IHtcbiAgICBcbn0pIl19