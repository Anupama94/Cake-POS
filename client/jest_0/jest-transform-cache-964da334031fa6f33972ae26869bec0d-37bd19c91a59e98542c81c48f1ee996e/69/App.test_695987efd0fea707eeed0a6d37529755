89d62d3b4f05243e6ed2be00ae990827
"use strict";

// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
//
// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
test('Fake Test', () => {
  expect(true).toBeTruthy();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbInRlc3QiLCJleHBlY3QiLCJ0b0JlVHJ1dGh5Il0sIm1hcHBpbmdzIjoiOztBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUdBQSxJQUFJLENBQUMsV0FBRCxFQUFjLE1BQU07QUFDcEJDLEVBQUFBLE1BQU0sQ0FBQyxJQUFELENBQU4sQ0FBYUMsVUFBYjtBQUNILENBRkcsQ0FBSiIsInNvdXJjZXNDb250ZW50IjpbIi8vIGltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG4vLyBpbXBvcnQgUmVhY3RET00gZnJvbSAncmVhY3QtZG9tJztcbi8vIGltcG9ydCBBcHAgZnJvbSAnLi9BcHAnO1xuLy9cbi8vIGl0KCdyZW5kZXJzIHdpdGhvdXQgY3Jhc2hpbmcnLCAoKSA9PiB7XG4vLyAgIGNvbnN0IGRpdiA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2RpdicpO1xuLy8gICBSZWFjdERPTS5yZW5kZXIoPEFwcCAvPiwgZGl2KTtcbi8vICAgUmVhY3RET00udW5tb3VudENvbXBvbmVudEF0Tm9kZShkaXYpO1xuLy8gfSk7XG5cblxudGVzdCgnRmFrZSBUZXN0JywgKCkgPT4ge1xuICAgIGV4cGVjdCh0cnVlKS50b0JlVHJ1dGh5KCk7XG59KSJdfQ==