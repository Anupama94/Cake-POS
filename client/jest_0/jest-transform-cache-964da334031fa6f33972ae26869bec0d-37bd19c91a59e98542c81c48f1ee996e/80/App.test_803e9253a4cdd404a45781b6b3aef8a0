bf2822b3e542e9c0e2ad5bc4f6615d30
"use strict";

var _interopRequireDefault = require("/home/anupamaa/Desktop/Training/pos/server/client/node_modules/@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _App = _interopRequireDefault(require("./App"));

var _jsxFileName = "/home/anupamaa/Desktop/Training/pos/server/client/src/App.test.js";
describe('App', () => {
  it('should render correctly in "debug" mode', () => {
    const component = (0, _enzyme.shallow)(_react.default.createElement(_App.default, {
      debug: true,
      __source: {
        fileName: _jsxFileName,
        lineNumber: 22
      },
      __self: void 0
    })); // expect(component).toMatchSnapshot();

    expect(component).toMatchSnapshot();
  });
  it('should render a anchor tag', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement("a", {
      className: "App-link",
      href: "https://reactjs.org",
      target: "_blank",
      rel: "noopener noreferrer",
      __source: {
        fileName: _jsxFileName,
        lineNumber: 30
      },
      __self: void 0
    }, "Learn React")); // expect(component).toMatchSnapshot();

    expect(wrapper).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbImRlc2NyaWJlIiwiaXQiLCJjb21wb25lbnQiLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiLCJ3cmFwcGVyIl0sIm1hcHBpbmdzIjoiOzs7O0FBZUE7O0FBQ0E7O0FBQ0E7OztBQUVBQSxRQUFRLENBQUMsS0FBRCxFQUFRLE1BQU07QUFDbEJDLEVBQUFBLEVBQUUsQ0FBQyx5Q0FBRCxFQUE0QyxNQUFNO0FBQ2hELFVBQU1DLFNBQVMsR0FBRyxxQkFBUSw2QkFBQyxZQUFEO0FBQUssTUFBQSxLQUFLLE1BQVY7QUFBQTtBQUFBO0FBQUE7QUFBQTtBQUFBO0FBQUEsTUFBUixDQUFsQixDQURnRCxDQUdqRDs7QUFDQ0MsSUFBQUEsTUFBTSxDQUFDRCxTQUFELENBQU4sQ0FBa0JFLGVBQWxCO0FBQ0gsR0FMQyxDQUFGO0FBT0FILEVBQUFBLEVBQUUsQ0FBQyw0QkFBRCxFQUErQixNQUFNO0FBQ25DLFVBQU1JLE9BQU8sR0FBRyxxQkFDWjtBQUNKLE1BQUEsU0FBUyxFQUFDLFVBRE47QUFFSixNQUFBLElBQUksRUFBQyxxQkFGRDtBQUdKLE1BQUEsTUFBTSxFQUFDLFFBSEg7QUFJSixNQUFBLEdBQUcsRUFBQyxxQkFKQTtBQUFBO0FBQUE7QUFBQTtBQUFBO0FBQUE7QUFBQSxxQkFEWSxDQUFoQixDQURtQyxDQVluQzs7QUFDQUYsSUFBQUEsTUFBTSxDQUFDRSxPQUFELENBQU4sQ0FBZ0JELGVBQWhCO0FBQ0gsR0FkQyxDQUFGO0FBZUgsQ0F2Qk8sQ0FBUiIsInNvdXJjZXNDb250ZW50IjpbIi8vIGltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG4vLyBpbXBvcnQgUmVhY3RET00gZnJvbSAncmVhY3QtZG9tJztcbi8vIGltcG9ydCBBcHAgZnJvbSAnLi9BcHAnO1xuLy9cbi8vIGl0KCdyZW5kZXJzIHdpdGhvdXQgY3Jhc2hpbmcnLCAoKSA9PiB7XG4vLyAgIGNvbnN0IGRpdiA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2RpdicpO1xuLy8gICBSZWFjdERPTS5yZW5kZXIoPEFwcCAvPiwgZGl2KTtcbi8vICAgUmVhY3RET00udW5tb3VudENvbXBvbmVudEF0Tm9kZShkaXYpO1xuLy8gfSk7XG5cblxuLy8gdGVzdCgnRmFrZSBUZXN0JywgKCkgPT4ge1xuLy8gICAgIGV4cGVjdCh0cnVlKS50b0JlVHJ1dGh5KCk7XG4vLyB9KVxuXG5pbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuaW1wb3J0IHsgc2hhbGxvdyB9IGZyb20gJ2VuenltZSc7XG5pbXBvcnQgQXBwIGZyb20gJy4vQXBwJztcblxuZGVzY3JpYmUoJ0FwcCcsICgpID0+IHtcbiAgICBpdCgnc2hvdWxkIHJlbmRlciBjb3JyZWN0bHkgaW4gXCJkZWJ1Z1wiIG1vZGUnLCAoKSA9PiB7XG4gICAgICAgIGNvbnN0IGNvbXBvbmVudCA9IHNoYWxsb3coPEFwcCBkZWJ1ZyAvPik7XG5cbiAgICAgICAvLyBleHBlY3QoY29tcG9uZW50KS50b01hdGNoU25hcHNob3QoKTtcbiAgICAgICAgZXhwZWN0KGNvbXBvbmVudCkudG9NYXRjaFNuYXBzaG90KCk7XG4gICAgfSk7XG5cbiAgICBpdCgnc2hvdWxkIHJlbmRlciBhIGFuY2hvciB0YWcnLCAoKSA9PiB7XG4gICAgICAgIGNvbnN0IHdyYXBwZXIgPSBzaGFsbG93KFxuICAgICAgICAgICAgPGFcbiAgICAgICAgY2xhc3NOYW1lPVwiQXBwLWxpbmtcIlxuICAgICAgICBocmVmPVwiaHR0cHM6Ly9yZWFjdGpzLm9yZ1wiXG4gICAgICAgIHRhcmdldD1cIl9ibGFua1wiXG4gICAgICAgIHJlbD1cIm5vb3BlbmVyIG5vcmVmZXJyZXJcIlxuICAgICAgICAgICAgPlxuICAgICAgICAgICAgTGVhcm4gUmVhY3RcbiAgICAgICAgPC9hPlxuICAgICAgICApO1xuXG4gICAgICAgIC8vIGV4cGVjdChjb21wb25lbnQpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICAgICAgICBleHBlY3Qod3JhcHBlcikudG9NYXRjaFNuYXBzaG90KCk7XG4gICAgfSk7XG59KTsiXX0=