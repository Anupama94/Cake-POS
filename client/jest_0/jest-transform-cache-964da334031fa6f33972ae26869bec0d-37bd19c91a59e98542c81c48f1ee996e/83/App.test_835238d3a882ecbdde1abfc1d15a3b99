eed57c54050ba90ac91004e535a49116
"use strict";

// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
//
// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
test('Fake Test');
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbInRlc3QiXSwibWFwcGluZ3MiOiI7O0FBQUE7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBR0FBLElBQUksQ0FBQyxXQUFELENBQUoiLCJzb3VyY2VzQ29udGVudCI6WyIvLyBpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuLy8gaW1wb3J0IFJlYWN0RE9NIGZyb20gJ3JlYWN0LWRvbSc7XG4vLyBpbXBvcnQgQXBwIGZyb20gJy4vQXBwJztcbi8vXG4vLyBpdCgncmVuZGVycyB3aXRob3V0IGNyYXNoaW5nJywgKCkgPT4ge1xuLy8gICBjb25zdCBkaXYgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdkaXYnKTtcbi8vICAgUmVhY3RET00ucmVuZGVyKDxBcHAgLz4sIGRpdik7XG4vLyAgIFJlYWN0RE9NLnVubW91bnRDb21wb25lbnRBdE5vZGUoZGl2KTtcbi8vIH0pO1xuXG5cbnRlc3QoJ0Zha2UgVGVzdCcpIl19