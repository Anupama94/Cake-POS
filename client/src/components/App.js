import React from 'react';

import { BrowserRouter, Route, Switch } from "react-router-dom";

import Login from "./Login/Login";
import OrderList from "./OrderList/OrderList";
import OrderDetailsView from "./OrderDetails/OrderDetailsView";
import ProtectedRoute from "../authentication/protectedRoutes";
import Page404 from './Page404';

class App extends React.Component {
    render() {
        return (
            <BrowserRouter>
                <Switch>
                    <Route path="/" component={Login} exact/>
                    <ProtectedRoute path="/OrderList" component={OrderList} exact/>
                    <ProtectedRoute path="/OrderDetails/:id" component={OrderDetailsView} exact/>
                    <Route path="" component={Page404} />
                </Switch>
            </BrowserRouter>
        );
    }
}

export default App;