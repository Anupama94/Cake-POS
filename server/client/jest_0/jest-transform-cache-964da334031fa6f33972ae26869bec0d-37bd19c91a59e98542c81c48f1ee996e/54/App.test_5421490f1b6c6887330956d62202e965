0f085ecb462155976fe7638e21bb04e6
// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
//
// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
"use strict";
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkFwcC50ZXN0LmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQSIsInNvdXJjZXNDb250ZW50IjpbIi8vIGltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG4vLyBpbXBvcnQgUmVhY3RET00gZnJvbSAncmVhY3QtZG9tJztcbi8vIGltcG9ydCBBcHAgZnJvbSAnLi9BcHAnO1xuLy9cbi8vIGl0KCdyZW5kZXJzIHdpdGhvdXQgY3Jhc2hpbmcnLCAoKSA9PiB7XG4vLyAgIGNvbnN0IGRpdiA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2RpdicpO1xuLy8gICBSZWFjdERPTS5yZW5kZXIoPEFwcCAvPiwgZGl2KTtcbi8vICAgUmVhY3RET00udW5tb3VudENvbXBvbmVudEF0Tm9kZShkaXYpO1xuLy8gfSk7XG5cblxuIl19