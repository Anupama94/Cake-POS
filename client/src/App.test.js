// import React from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
//
// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });


// test('Fake Test', () => {
//     expect(true).toBeTruthy();
// })

import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<App debug />);

       // expect(component).toMatchSnapshot();
        expect(component).toMatchSnapshot();
    });

    it('should render a anchor tag', () => {
        const wrapper = shallow(
            <a
        className="App-link"
        href="https://reactjs.org"
        target="_blank"
        rel="noopener noreferrer"
            >
            Learn React
        </a>
        );

        // expect(component).toMatchSnapshot();
        expect(wrapper).toMatchSnapshot();
    });
});