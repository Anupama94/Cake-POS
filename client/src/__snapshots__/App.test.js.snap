// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render a anchor tag 1`] = `
<a
  className="App-link"
  href="https://reactjs.org"
  rel="noopener noreferrer"
  target="_blank"
>
  Learn React
</a>
`;

exports[`App should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
